roles/provision-buildfarm-apb/tasks/artifact-proxy-operator.yml

```diff
@@ -54,7 +54,7 @@
         value: "{{ proxy_route_host.stdout }}"
       - name: JENKINS_HOSTNAME
         value: "{{ jenkins_route_host.stdout }}"
-      image: 'docker.io/aerogear/artifact-proxy-operator:latest'
+      image: 'docker.io/aerogear/artifact-proxy-operator:1.0.0-alpha'
```

Comment:
@philbrookes can you ping when this tag is available

Reply:
Just ran `docker pull  docker.io/aerogear/artifact-proxy-operator:1.0.0-alpha` and the image was pulled.